Media/index.html

```diff
@@ -0,0 +1,679 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://unicons.iconscout.com/release/v2.1.6/css/unicons.css">
+    <title>Media Social</title>
+    <!-- StyleSheet -->
+    <link rel=" stylesheet" href="./style.css">
+</head>
+
+<body>
+    <nav>
+        <div class="container">
+            <h2 class="log">
+                Media Social
+            </h2>
+            <div class="search-bar">
+                <i class="uil uil-search">
+                    <input type="search" placeholder="Search for creators,">
+                </i>
+            </div>
+            <div class="create">
+                <label for="create-post" class="btn btn-primary">Create</label>

```

Comment:
Thuộc tính for trong label có ý nghĩa gì? Nếu bỏ for đi trong trường hợp này thì có vấn đề gì sảy ra?

Reply:
thuộc tính for trong label là nhằm giúp cho thẻ label có thể kết nối với thẻ input 
trong trường hợp này nếu xoá đi thì nó không ảnh hướng và ko có vấn đề gì xảy ra